### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by 1-byte-man**
Spanish translation of the homepage

### CODE REVIEW

**Comment by 1-byte-man** [comment]
Ok it's done

**Comment by shushugah** [comment]
Thank you for this! There is one last step, and a semi related blocker. 

1. You will need to add `"es"` to this line as well. I would be fine with making this the second language. If you could update the README.md as well, where it says that Spanish is a supported language that would be great.  https://github.com/techworkersco/twc-site/blob/develop/_config.yml#L32 

2. This is not specific to your Spanish translation PR, but right now, a 4th language breaks the UI, because they do not fight nicely together. A dropdown style menu would work.

**Comment by 1-byte-man** [comment]
> 
> 
> You will need to add "es" here: would be great. https://github.com/techworkersco/twc-site/blob/develop/_config.yml#L32 something like `languages: ["en", "es", "ru", "it"]`
> 
> @1-byte-man this still needs to be implemented

yes it's https://github.com/1-byte-man/twc-site/blob/es-translation/_config.yml

**Comment by ergl** [comment]
Hi! there are a few typos in here, but overall it looks good. I re-worded some things, take a look here https://gist.github.com/ergl/cb88f1b963d9b9216d1cc9c0788dde8d

The term "rank & file" is hard to translate, so I decided to omit it and focus on the worker power angle.

**Comment by shushugah** [comment]
> > You will need to add "es" here: would be great. https://github.com/techworkersco/twc-site/blob/develop/_config.yml#L32 something like `languages: ["en", "es", "ru", "it"]`
> > @1-byte-man this still needs to be implemented
> 
> yes, there's https://github.com/1-byte-man/twc-site/blob/es-translation/_config.yml

This is in a different branch. You made that commit in branch `es-translations` which you can see here https://github.com/1-byte-man/twc-site/commits/es-translation

but you want to cherry-pick the branch into this current branch which is `https://github.com/1-byte-man/twc-site/commits/spanish-translation` I checked and that was the only thing left

**Comment by 1-byte-man** [comment]
ok now it's done https://github.com/1-byte-man/twc-site/blob/develop/_config.yml

**Comment by shushugah** [comment]
Right now 4 languages would break the UI in mobile view, so my suggestion is to implement something like this. After that's merged, then we can merge this
https://codyhouse.co/blog/post/accessible-language-picker

File: _i18n/es.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# 1.0.0
+langs:
+  es: Español
+home:
+  title: |
+    <span>Poder</span> para los trabajadores en la industria de la tecnología
+    summary: Guiados por nuestra visión para una industria de la teconogía mas equitativa e inclusiva, TWC se organiza para construir un poder para los trabajadores mediante la educación y autoorganizacion
```
Comment (alvaromontoro) [inline _i18n/es.yml]: Should this line be less indented? (I cannot test it at the moment)

Comment (1-byte-man) [reply _i18n/es.yml]: me too but I didn't change the identation. So I think it should work

**Comment by ergl** [comment]
I see the language dropdown is now in the page. If no one is against it, I'll take up the task of finishing the spanish translation

**Comment by shushugah** [comment]
Shall I resolve the conflicts for you, or will you manage?

File: _i18n/es/home.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<div class="rich-text home-page">
```
Comment (shushugah) [inline _i18n/es/home.html:1]: Also this entire page can be deleted, the `index.md` purely uses the translation keys now

**Comment by ergl** [comment]
I think it's fine! I'll let you know if I encounter any problems

**Pull request closed without merging**